docs/projects/adv7513/index.rst

```diff
@@ -0,0 +1,180 @@
+.. _adv7513:
+
+ADV7513 HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The :adi:`ADV7513` is a 165 MHz, High-Definition Multimedia Interface (HDMI®)
+transmitter that is ideal for DVD players/ recorders, digital set-top boxes,
+A/V receivers, gaming consoles, and PCs.
+
+The digital video interface contains an HDMI v1.4/DVI v1.0-compatible transmitter
+and supports all HDTV formats. The :adi:`ADV7513` supports HDMI v1.4-specific features,
+including 3D video. The :adi:`ADV7513` also supports x.v.Color™, high bit rate (HBR) audio,
+and the programmable auxiliary video information (AVI) InfoFrame features.
+With the inclusion of HDCP, the :adi:`ADV7513` allows the secure transmission of protected content
+as specified by the HDCP v1.4 protocol.
+
+The :adi:`ADV7513` supports both S/PDIF and 8-channel I2S audio. Its high fidelity 8-channel
+I2S interface can transmit either stereo or 7.1 surround audio up to 768 kHz. The S/PDIF interface
+can carry compressed audio, including Dolby® Digital, DTS®, and THX®. Fabricated in an advanced CMOS process,
+the :adi:`ADV7513` is provided in a 64-lead LQFP surface-mount plastic package with exposed pad
+and is specified over the −25°C to +85°C temperature range.
+
```

Comment:
Could you wrap the text in the Overview section, to have the same maximum characters no./line? Use the dashed lines as reference.